### PR TITLE
MODORGS-34 Create release 2.0.0

### PR DESCRIPTION
**Pull request by grabsefx**
## Purpose
https://issues.folio.org/browse/MODORGS-34 Create MODORGS release

## Approach
* Interface versions updated
* News.md file updated

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     },
     {
       "id": "organizations-storage.contacts",
-      "version": "1.1",
+      "version": "2.0",
```
Comment (pautau-s) [inline descriptors/ModuleDescriptor-template.json]: Why it should be bumped up to 2.0? I can't find any breaking changes for contacts

Comment (grabsefx) [reply descriptors/ModuleDescriptor-template.json]: `contact.json` has a reference to `phone_number.json` schema which was significantly changed. So, contacts interface no more compatible with version 1.x

Comment (pautau-s) [reply descriptors/ModuleDescriptor-template.json]: Didn't notice, thank you for pointing it out.

**Comment by KVupp** [comment]
I just checked that the fromModuleVersion is for the version the table was updated in too.
> fromModuleVersion - this field indicates the version in which the table was created / updated in.
I suppose as we added metadata property to the tables, the schema has to be updated to latest version for all the tables?

**Comment by pautau-s** [comment]
> I suppose as we added metadata property to the tables, the schema has to be updated to latest version for all the tables?

Actually, the metadata was present in the tables originally. But during the migration to RMB 25, `generateId` and `populateJsonWithId` properties were removed.

**Comment by khandramai** [comment]
Hi @KVupp,
according RAML documentation:

> fromModuleVersion - this field indicates the version in which the table was created / updated in. When a tenant update is requested - only versions older than the indicated version will generate the declared table. This ensures that if a module upgrades from an older version, the needed tables will be generated for it, however, subsequent upgrades from versions equal or later than the version indicated for the table will not re-generate the table.

In my understanding application of this parameter is:

> This ensures that if a module upgrades from an older version, the needed tables will be generated for it, however, subsequent upgrades from versions equal or later than the version indicated for the table will not re-generate the table.

As @pautau-s mentioned that metadata was presented in the tables initially, so maybe it makes sense to leave it as is - this guarantee that table will not be re-generated.

**Comment by KVupp** [comment]
@pautau-s / @khandramai , Thanks for pitching in , but I was referring to the meta data that was added to the schemas(which would then be populated by the "withMetadata" trigger, which was already enabled on the tables), with the story https://issues.folio.org/browse/MODORGS-32.
Also like @khandramai  mentioned, RMB had breaking changes.

**Pull request merged**